tools/install_demo_configuration.sh

```diff
@@ -425,6 +425,7 @@ else
 	echo "### or run ./securityadmin_demo.sh"
 	echo "### To use the Security Plugin ConfigurationGUI"
 fi
+eifjcbdkithivclitcurdbgnrcuhjfhffjdrbghfjelu
```

Comment:
Looks like this inadvertently got in here.

Reply:
Oops haha. Thank you!